sdk/python/foundation-models/llama2/promptFlow/Chat_flow_sample/chatting_with_Gitdoc/readme.md

```diff
@@ -0,0 +1 @@
+This is chat flow sample, where you can pass url and model will reply your's desired questions answer related to that url.
```

Comment:
Replace "your's desired questions" for "your input questions"

Reply:
Done.